### PR TITLE
Added student's avatars for beta + fixed rating and profile pages + tests

### PR DESCRIPTION
**Pull request by ElyDev**

### CODE REVIEW

File: Dev/QConnect/src/components/ImageSelectionModal.js [hunk 1]
```diff
@@ -20,14 +20,15 @@ export default ImageSelectionModal = (props) => {
...
-            <View style={{ flexDirection: 'row', flexWap: 'wrap', justifyContent: 'center', alignItems: 'center' }}>
+            <View style={{ flexDirection: 'row', flexWap: 'wrap', justifyContent: 'center', alignItems: 'center', marginBottom: 10 }}>
                 <View style={styles.container}>
                     <ImageSelectionGrid
                         images={props.images}
                         onImageSelected={props.onImageSelected}
                     />
                     <TouchableText
                         text={props.cancelText}
+                        style={{fontSize: 14, marginTop: 10 }}
```
Comment (zyadelgohary) [inline Dev/QConnect/src/components/ImageSelectionModal.js]: Add fontFamily: 'regular"

File: Dev/QConnect/src/components/TouchableText.js [hunk 2]
```diff
@@ -19,11 +19,11 @@ const styles = StyleSheet.create({
     container: {
         color: colors.primaryDark,
         marginLeft:10,
-        marginTop:10,
         marginBottom:10,
         paddingTop:7,
         paddingBottom:7,
         paddingRight:21,
+        fontSize: 11,
```
Comment (zyadelgohary) [inline Dev/QConnect/src/components/TouchableText.js]: We definitely need to have a set fontSize and fontStyle class similar to how I made a strings class.

File: Dev/QConnect/src/screens/Evaluation/EvaluationPage.js [hunk 3]
```diff
@@ -64,6 +39,7 @@ export class EvaluationPage extends Component {
   // --------------  Renders Evaluation scree UI --------------
   render() {
     const { classIndex, studentIndex } = this.props.navigation.state.params;
+    const { imageId } = this.props;
```
Comment (zyadelgohary) [inline Dev/QConnect/src/screens/Evaluation/EvaluationPage.js]: Shouldn't this be " const { imageId } = this.props.classes[classIndex].students[studentIndex] "?

File: Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js [hunk 4]
```diff
@@ -26,88 +29,149 @@ class StudentProfileScreen extends Component {
...
   }
 
+  //---------- profile image views handlers --------------
+  setModalVisible(visible) {
+    this.setState({ isModalVisible: visible });
+  }
+
+  //this method saves the new profile information to the redux database
+  saveProfileInfo = (teacherID) => {
+
```
Comment (zyadelgohary) [inline Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js]: Does this method still need to be created?

File: Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js [hunk 5]
```diff
@@ -117,52 +181,107 @@ class StudentProfileScreen extends Component {
...
   bigText: {
     fontSize: 24,
-    fontFamily: fonts.regular
+    fontFamily: 'regular',
   },
   subText: {
+    fontSize: 16,
+    fontFamily: 'regular',
+    color: colors.primaryDark
+  },
```
Comment (zyadelgohary) [inline Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js]: This is what I meant... if we can just move these to a separate class (the above 2 styles) & use them throughout the app, that'd be great. Next PR isA.

File: Dev/QConnect/src/components/ImageSelectionModal.js (hunk 1 above)
Comment (ElyDev) [reply Dev/QConnect/src/components/ImageSelectionModal.js]: Added to the TouchableText style itself. This passed in style is a override for properties we want to modify.

File: Dev/QConnect/src/screens/Evaluation/EvaluationPage.js (hunk 3 above)
Comment (ElyDev) [reply Dev/QConnect/src/screens/Evaluation/EvaluationPage.js]: not anymore. I changed mapStateToProps to only return the current student.

const mapStateToProps = (state, ownProps) => {
  const { classIndex, studentIndex } = ownProps.navigation.state.params;
  state = state.data.teachers[0].classes[classIndex].students[studentIndex];
  return state;
};

File: Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js (hunk 5 above)
Comment (ElyDev) [reply Dev/QConnect/src/screens/StudentProfile/StudentProfileScreen.js]: Agree..

**Pull request merged**